app/pulp/app/models/progress.py

```diff
@@ -18,6 +20,16 @@ class ProgressReport(Model):
 
     All progress reports have a message, state, and are related to a Task.
 
+    Attributes:
+
+        batch_increments (bool): If this Flag is set save checks to make sure it only gets ran once
```

Comment:
I don't think practically speaking a plugin writer would ever want to disable this feature. If there was a reason they would then I would want to add it. My main concern is that we want to launch with a really really simple plugin API and these docstring additions are making it more complicated with an option that I think almost no one will use.